### PR TITLE
[IMP] doc: improve wording in profile docs

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Improve the documentation

Current behavior before PR: Some sentences didn't read very nicely.

Desired behavior after PR is merged: Nice sentences!

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/howtos/profilecode.rst [hunk 1]
```diff
@@ -78,16 +78,16 @@ completely reviewed.
...
 odoo process seems stucked, sending this signal to the process permit to know
 what the process is doing, and letting the process continue his job.
 
 Tracing code execution
 ======================
 
-Instead of sending the SIGQUIT signal to an odoo process often enough, to check
-where processes is performing worse than expected, we can use pyflame tool to
+Instead of sending the SIGQUIT signal to an Odoo process often enough, to check
+where the processes is performing worse than expected, we can use pyflame tool to
```
Comment (pedrobaeza) [inline doc/howtos/profilecode.rst]: s/is/are

File: doc/howtos/profilecode.rst [hunk 2]
```diff
@@ -118,7 +118,7 @@ Once done, we'll display them as an execution graph.
 
 where <PID> is the process ID of the odoo process you want to graph. This will
 wait until the dead of the process, with a maximum of one hour, and and get 5
-traces a second. With the output of pyflame, we can produce an svg graph with
+traces a second. With the output of pyflame, we can produce an SVG graph with
```
Comment (mreficent) [inline doc/howtos/profilecode.rst]: s/an SVG/a SVG?

Comment (Yenthe666) [reply doc/howtos/profilecode.rst]: Hmm I'm not absolutely sure honestly. 😆

**Comment by Yenthe666** [comment]
Thanks and welcome back Martin!

**Pull request closed without merging**